### PR TITLE
REC-127 | added datasets & minor style fixes

### PR DESCRIPTION
**Pull request by anastasissimo**

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -14,10 +15,14 @@ function App() {
           path="/search"
           render={routeProps => <SearchResultPage query={routeProps.location.search} />}
         />
-        <Route path="/">
+        <Route exact={true} path="/">
```
Comment (dkrupenya) [inline src/App.tsx]: You don't need exact
You just need to move `<Route path="/files">` above `<Route path="/">`
Routes are matched in the order of appearance

File: src/App.tsx [hunk 2]
```diff
@@ -14,10 +15,14 @@ function App() {
...
           {/*<SearchPage />*/}
           <SearchResultPage query="" />
         </Route>
+        <Route exact={true} path="/files">
+          {/*<SearchPage />*/}
```
Comment (dkrupenya) [inline src/App.tsx]: Remove

File: src/pages/FilesPage/FilesPage.tsx [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+import { SearchResultSidebar } from '../SearchResultPage/SearchResultSidebar/SearchResultSidebar';
+
+import { FileSearch } from './FileSearch/FileSearch';
+import style from './FilesPage.module.scss';
+import { FilesTabs } from './FilesTabs/FilesTabs';
+
+export const FilesPage: React.FC = function FilesPage() {
+  return (
+    <div className={style.root}>
+      <span className={style.main}>
```
Comment (dkrupenya) [inline src/pages/FilesPage/FilesPage.tsx]: div can't be child of span
replace with div

File: src/pages/FilesPage/FilesTabs/DatasourcesTable/DatasourcesTable.tsx [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+import { ReactComponent as MoreIcon } from '../../../../resources/more.svg';
+import { ArticlesListItem } from '../../../SearchResultPage/SearchResultMain/ArticlesList/ArticlesListItem/ArticlesListItem';
+
+import style from './DatasourcesTable.module.scss';
+
+type DatasourcesProps = {
+  datasources: IDatasourceResponseObject[];
+};
+
+export const DatasourcesTable: FC<DatasourcesProps> = function ArticlesList({
```
Comment (dkrupenya) [inline src/pages/FilesPage/FilesTabs/DatasourcesTable/DatasourcesTable.tsx:15]: General comment. Later this component should be splited into several ones

File: src/pages/FilesPage/FilesTabs/DatasourcesTable/DatasourcesTable.tsx [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+  datasources,
+}) {
+  return (
+    <div className={style.root}>
+      <table className={style.ul}>
```
Comment (dkrupenya) [inline src/pages/FilesPage/FilesTabs/DatasourcesTable/DatasourcesTable.tsx:20]: To be rewritten with Antd table later

File: src/pages/FilesPage/FilesTabs/FilesTabs.module.scss [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+}
+
+.root {
+  padding-top: 32px;
+}
+
+.paginator{
+  position: absolute;
+  bottom: 0;
+  width: calc(100% - 66px);
```
Comment (dkrupenya) [inline src/pages/FilesPage/FilesTabs/FilesTabs.module.scss:30]: magic.

File: src/pages/FilesPage/FilesTabs/FilesTabs.module.scss [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+  display: flex;
+  justify-content: center;
+  padding-bottom: 69px;
```
Comment (dkrupenya) [inline src/pages/FilesPage/FilesTabs/FilesTabs.module.scss:33]: Typically paddings are multipliers of 4

File: src/pages/FilesPage/FilesTabs/FilesTabs.tsx [hunk 8]
```diff
@@ -0,0 +1,73 @@
...
+              <DatasourceIcon />
+              Datasources
+            </span>
+          }
+          key="1"
+        >
+          <DatasourcesTable datasources={mockedDatasource} />
+          <div className={style.paginator}>
+            <Paginator />
+          </div>
```
Comment (dkrupenya) [inline src/pages/FilesPage/FilesTabs/FilesTabs.tsx:35]: This should be a separate component because paginator belongs to table, not to tabs

**Review by dkrupenya — APPROVED** [review]

File: src/pages/FilesPage/FilesTabs/FilesTabs.module.scss (hunk 6 above)
Comment (anastasissimo) [reply src/pages/FilesPage/FilesTabs/FilesTabs.module.scss:30]: this is because of outer paddings

File: src/App.tsx (hunk 1 above)
Comment (dkrupenya) [reply src/App.tsx]: fixed

File: src/App.tsx (hunk 2 above)
Comment (dkrupenya) [reply src/App.tsx]: done

File: src/pages/FilesPage/FilesPage.tsx (hunk 3 above)
Comment (dkrupenya) [reply src/pages/FilesPage/FilesPage.tsx]: done

**Pull request merged**